ludwig/schema/features/base.py

```diff
@@ -44,19 +43,6 @@ class BaseFeatureConfig(schema_utils.BaseMarshmallowConfig):
         description="Type of the feature.",
     )
 
```

Comment:
These params along with all others that are internal only are getting removed off the `ModelConfig` object. I may have missed some internal parameters though so feel free to identify any others that I may need to add to the metadata object somewhere. I.e. category decoder `num_classes`?